src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -82,6 +85,25 @@ export class DBTCoreProjectDetection
     private executionInfrastructure: DBTCommandExecutionInfrastructure,
   ) {}
 
+  private getPackageInstallPathFallback(projectDirectory: Uri): string {
+    const dbtProjectFile = path.join(
+      projectDirectory.fsPath,
+      "dbt_project.yml",
+    );
+    if (existsSync(dbtProjectFile)) {
+      const dbtProjectConfig: any = parse(readFileSync(dbtProjectFile, "utf8"));
+      const packagesInstallPath = dbtProjectConfig["packages-install-path"];
+      if (packagesInstallPath) {
+        if (path.isAbsolute(packagesInstallPath)) {
+          return packagesInstallPath;
+        } else {
+          return path.join(projectDirectory.fsPath, packagesInstallPath);
+        }
+      }
+    }
+    return path.join(projectDirectory.fsPath, "dbt_packages");
```

Comment:
On line 109 we already do the same. I think we don't need to return this. and just only do the work if the fallback returns something.